src/ratings-and-reviews/ratings-and-reviews.scss

```diff
@@ -76,4 +76,46 @@
   .star-breakdown:hover {
     text-decoration: underline;
   }
+
+  .star-rating-buttons {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+  }
+  .star-rating-buttons input{
+    display: none;
+  }
+  .star-rating-buttons label {
+    width: 20px;
+    height: 20px;
+    font-family: Verdana;
+    font-size: 20px;
+    color: black;
+    transition: 0.2s ease;
+  }
+  .star-rating-buttons label::before {
+    content: '\2606';
+  }
+  .star-rating-buttons input:checked ~ label:before {
```

Comment:
Okay, this is very cool. I had to look up what it was doing and I'm still not entirely sure.